### ISSUE TITLE
### nb of giveaway foud  0

### ISSUE THREAD

**Issue by GridinoEpic** [steevenakintilo/TwitterGiveawayBot#13]
I can get to the point where it searches for my giveaway words and scrolls down, but the console says : ### nb of giveaway foud  0
Even tho there are giveaways on the page there. Why is that?

**Comment by steevenakintilo** [comment]
Yeah saw that today probably a twitter problem fixed it with setting nb_of_search_giveaway to 1000 so it never ran of ammount

**Comment by GridinoEpic** [comment]
> Yeah saw that today probably a twitter problem fixed it with setting nb_of_search_giveaway to 1000 so it never ran of ammount

I tried the new fix but it still has the same issue

**Comment by GridinoEpic** [comment]
![asdasdas](https://github.com/user-attachments/assets/987140c2-0344-49c7-9624-6948c980bed7)

**Comment by steevenakintilo** [comment]
Oh wait try the same thing but with one word per search may be that

**Comment by GridinoEpic** [comment]
Still nothing. I even tried your original file (the one in french) and it doesnt work.

**Comment by steevenakintilo** [comment]
Strange because I had the same issue as you and re run the bot after and got no issue

**Comment by steevenakintilo** [comment]
let me check again

**Comment by GridinoEpic** [comment]
One thing i have to say is that it didnt seem to install things with python 3.12 so i had to run a virtual enviroment with python 3.10 which does seem to work, until yeah the no giveaway found

**Comment by steevenakintilo** [comment]
wait so it's working with python 3.10 but withouth 3.12?

**Comment by GridinoEpic** [comment]
Yeah it didnt seem to install the requirements with python 3.12

**Comment by GridinoEpic** [comment]
![image](https://github.com/user-attachments/assets/e0a4222f-532b-404b-98d5-85e28d5367de)
![image](https://github.com/user-attachments/assets/3a03be5f-22e2-4cea-ac33-f3356f57219c)
The errors it kept giving me

**Comment by steevenakintilo** [comment]
Oh that's weird well never ran the bot with python higher than 3.10 may be that

**Comment by GridinoEpic** [comment]
> Oh that's weird well never ran the bot with python higher than 3.10 may be that

But with the virtual enviroment it does seem to work, but yeah cant find any giveaways. I sent you a friend request on discord if you wanna discuss it over there

**Comment by steevenakintilo** [comment]
ok thanks let me accept
